oap-server/server-library/library-util/src/test/java/org/apache/skywalking/oap/server/library/util/StringUtilTest.java

```diff
@@ -76,4 +76,12 @@ public void testTrim() {
         assertEquals(StringUtil.trim("abcdef", 'a'), "bcdef");
         assertEquals(StringUtil.trim("abcdef", 'f'), "abcde");
     }
+
+    @Test
+    public void testTrimJson() {
+        String jsonString = "{\"k1\":\"v1\",\"k2\":\"v2\",\"k3\":true,\"k4\":[{\"a\":\"b\",\"c\":\"d\"},{\"e\":\"f\",\"g\":\"h\"}]}";
+        assertEquals(StringUtil.trimJson(jsonString, 10), "{\"k1\":\"v1\",\"k2\":\"v2\"}");
+        assertEquals(StringUtil.trimJson(jsonString, 30), "{\"k1\":\"v1\",\"k2\":\"v2\",\"k3\":true}");
+    }
```

Comment:
I think we should verify whether the size of the trimmed JSON str oversized.

Reply:
Sorry, I was careless.